### PR TITLE
First readme file

### PR DESCRIPTION
**Pull request by milan-panta**
# Description

Initial README.md file motivation, features, walkthrough and getting started.

Fixes # (issue)

## Type of change

- [] Bug fix (non-breaking change which fixes an issue)
- [X] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

# Checklist:

- [X] My code follows the style guidelines of this project
- [X] I have performed a self-review of my own code
- [X] I have commented my code, particularly in hard-to-understand areas

### CODE REVIEW

**Review by ji24077 — APPROVED** [review]
Looks even better than before Habibi :)

**Comment by ji24077** [comment]
This is good just like birryanni :)

File: README.md [hunk 1]
```diff
@@ -1,39 +1,64 @@
-# Project Template
+# AutoTrace
 
-This is a template repository for CSC 207 projects. 
-This repository contains starter code for a gradle project.
-It also contains workflow documents that give instructions on how to manage your Github repository and how to use Github Projects for efficient collaboration.
+AutoTrace is an Android app designed to provide users with essential information about vehicles by inputting their license plate numbers. It offers a simple and efficient way to retrieve car model and registration date data from relevant databases. Additionally, it provides administrators with the ability to register license plates into the system, along with associated driver and car information.
```
Comment (l0minous) [inline README.md:3]: AutoTrace is an Android app designed to provide users with essential vehicle information by inputting their license plate numbers. It offers a simple and efficient way to retrieve car model and registration date data from relevant databases. Additionally, it allows administrators to register license plates into the system, along with associated driver and car information.

File: README.md [hunk 2]
```diff
@@ -1,39 +1,64 @@
...
-- [ ] Verify the correct settings for your project repository
-- [ ] Set up Github Projects
-- [ ] Create the implementation plan using issues and Github Projects
-- [ ] Create deveopment branches for your features
-- [ ] Use pull requests to merge finished features into main branch
-- [ ] Conduct code reviews
+## Motivation
 
-**If your team has trouble with any of these steps, please ask on Piazza. For example, with how GitHub Classroom works, your team *may* not have permissions to do some of the first few steps, in which case we'll post alternative instructions as needed.**
+The primary motivation behind AutoTrace is to offer a convenient solution for users who need to identify car models and registration dates based on license plate numbers. This app aims to streamline the process of accessing vehicle information by leveraging databases and providing an intuitive user interface.
```
Comment (l0minous) [inline README.md:7]: We aimed to create an app that works as a centralized system for displaying issued traffic tickets. The app displays information about each car, such as insurance provider. Another app feature is verifying ownership details by looking up any vehicle registered by the license plate in the database. Users can create an account to register their cars and view filed traffic tickets.

File: README.md [hunk 3]
```diff
@@ -1,39 +1,64 @@
...
-The starter code was built using SDK version 11.0.1. Ensure that you are using this version for this project. (You can, of course, change the SDK version as per your requirement if your team has all agreed to use a different version)
+### License Plate Search
+1. User opens the AutoTrace Android app and launches the license plate search feature.
+2. They enter the license plate number into the provided input field and tap on the "Search" button.
+3. The app processes the input and queries the relevant databases based on the user's access key.
+4. The app retrieves the car model and registration date associated with the provided license plate number.
+5. Finally, the app displays the car model and registration date on the user interface, allowing the user to view the information.
 
-You have been provided with two starter files for demonstration: HelloWorld and HelloWorldTest.
+### Vehicle Registration (Admin)
```
Comment (l0minous) [inline README.md:26]: Delete, Admin?

**Review by l0minous — CHANGES_REQUESTED** [review]

File: README.md [hunk 4]
```diff
@@ -1,39 +1,64 @@
...
-- [ ] Create deveopment branches for your features
-- [ ] Use pull requests to merge finished features into main branch
-- [ ] Conduct code reviews
+## Motivation
 
-**If your team has trouble with any of these steps, please ask on Piazza. For example, with how GitHub Classroom works, your team *may* not have permissions to do some of the first few steps, in which case we'll post alternative instructions as needed.**
+The primary motivation behind AutoTrace is to offer a convenient solution for users who need to identify car models and registration dates based on license plate numbers. This app aims to streamline the process of accessing vehicle information by leveraging databases and providing an intuitive user interface.
 
-## Workflow Documents
+## Features
```
Comment (l0minous) [inline README.md:9]: Mention the Navigation Bar as a feature

**Review by l0minous — CHANGES_REQUESTED** [review]

**Review by l0minous — APPROVED** [review]

**Pull request merged**